packages/webapp/public/locales/en/translation.json

```diff
@@ -1046,6 +1050,8 @@
       "ABANDON_NOTES": "Abandonment notes",
       "ABANDON_PLAN": "Abandon plan",
       "ABANDON_REASON": "Reason for abandoning",
+      "CANT_COMPLETE_ABANDONED": "Can't complete crop plan",
+      "CANT_COMPLETE_CONCURRENT_USER": "Another farmer has abandoned this plan. Please refresh the page to get the most recent status.",
```

Comment:
```suggestion
      "CANT_COMPLETE_CONCURRENT_USER": "Another farmer has abandoned this plan. Please return to the list of crop plans to get the most recent status.",
```

It's a minor point, but on the page where this modal appears (unlike the page where the user tries to delete a plan), there is no indication of status that would change if they refreshed the page.

From this view the user needs to go back to the list of crop plans (and actually that will be enough -- refreshing is not necessary) to get the status to update.

![Screenshot 2023-11-15 at 12 18 17 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/4964f2c1-55a5-43be-bb35-2c79b2e7c49d)

Reply:
Thanks for pointing this out! I updated the copy 🙂